core/templates/components/state-directives/rule-editor/rule-editor.directive.ts

```diff
@@ -265,6 +271,21 @@ angular.module('oppia').directive('ruleEditor', [
           };
 
           ctrl.$onInit = function() {
+            ctrl.isInvalid = false;
+            if (ctrl.isEditingRuleInline()) {
```

Comment:
Can you please explain why this condition is required?

Reply:
Added a comment above the if condition.

Reply:
Thanks for explaining!